packages/@react-spectrum/s2/stories/CardView.stories.tsx

```diff
@@ -112,7 +114,7 @@ function PhotoCard({item, layout}: {item: Item, layout: string}) {
   );
 }
 
-export const Example = (args: CardViewProps<any>) => {
+const ExampleRender = (args: CardViewProps<any>) => {
```

Comment:
Noticed that the ActionBar Card story is broken, maybe this needs to be imported instead?

Reply:
Thanks, fixed. That is awful that the inferred types don't catch that one